tools/draw-importances.py

```diff
@@ -1,3 +1,5 @@
+#!/usr/bin/python3
```

Comment:
```suggestion
#!/usr/bin/env python
```

This should work even if you have python on a special path (quite common)